scripts/createShadowCpp/createShadow.cpp

```diff
@@ -162,7 +170,49 @@ namespace shadow_graph {
             }
             this->nodes=nodes - 1;
             reader.close();
+            //writeBin(*this, filename);
+        }
+        //serialization
+        
+        friend class cereal::access;
+        template<class Archive>
+        void serialize(Archive& archive)
```

Comment:
This looks good, if I read correctly we have separte serialized graphs for the createShadow, right?

Reply:
Ya, but need to talk to you next week about it before I enable this in the repo since I haven't written the logic behind the loading yet.